src/main/java/com/commercetools/paymenttoorderprocessor/jobs/actions/MessageReader.java

```diff
@@ -127,7 +135,20 @@ private void fetchUnprocessedMessagesFromPlatform() {
     private MessageQuery buildQuery() {
 
         MessageQuery messageQuery = MessageQuery.of()
-                .withPredicates(m -> m.type().is(MESSAGETYPE))
+                .plusPredicates(m -> {
+                    QueryPredicate<Message> query = null;
```

Comment:
Maybe better to call it `predicate` and not a `query` since we already have a `messageQuery`?

Reply:
https://github.com/commercetools/commercetools-payment-to-order-processor/pull/71/commits/826e6c0bb784edc56638d6278d4b62f2e2bcc618